local-dev.sh

```diff
@@ -702,6 +703,12 @@ setup_database() {
 
 # Start development server
 start_dev_server() {
+    log_info "Generating Prisma client..."
+
+    cd "$DB_DIR"
+    pnpm run generate || error_exit "Failed to generate Prisma client"
```

Comment:
this was required for you? I thought I fixed it by adding generate to db package `prebuild` :(

Reply:
Seemed like it - failed first time - added this and it worked.. :(